.github/workflows/ci.yml

```diff
@@ -24,3 +24,21 @@ jobs:
       - run: |
           npm i
           npm run lint
+
+
+name: layer5-ng deploy
+on:
+  push:
+    branches: [layer5-ng]
+jobs:
+  deploy-GH:
+    name: Deploy on GH pages
+    runs-on: ubuntu-latest
```

Comment:
Can we add an information here "Ubuntu Latest" ?

Reply:
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md